client/components/global-notices/index.jsx

```diff
@@ -103,7 +104,7 @@ const NoticesList = React.createClass( {
 export default connect(
 	state => {
 		return {
-			storeNotices: state.notices.items
+			storeNotices: getNotices( state )
 		};
 	},
 	dispatch => bindActionCreators( { removeNotice }, dispatch )
```

Comment:
We can also pass an object `{ removeNotice }` here if we don't need to do anything custom.